app/api/votes/route.ts

```diff
@@ -12,8 +13,11 @@ export async function POST(request: Request) {
       args: [formation_id, userId],
     });
 
+    revalidatePath(`/formations/${formation_id}`);
+
     return new Response(null, { status: 204 });
   } catch (e) {
+    console.error(e);
```

Comment:
You meant to leave this in here?

Reply:
still a draft, i wanted to see how this ran in preview deploy. will be removing